Sources/Model/TransferMethod/HyperwalletPaperCheckAccount.swift

```diff
@@ -0,0 +1,513 @@
+//
+// Copyright 2018 - Present Hyperwallet
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software
+// and associated documentation files (the "Software"), to deal in the Software without restriction,
+// including without limitation the rights to use, copy, modify, merge, publish, distribute,
+// sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all copies or
+// substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import Foundation
+
+/// Representation of the user's paper check
+@objcMembers
+public class HyperwalletPaperCheckAccount: HyperwalletTransferMethod {
+    override private init(data: [String: AnyCodable]) {
+        super.init(data: data)
+    }
+    /// The required initializer
+    public required init(from decoder: Decoder) throws {
+        try super.init(from: decoder)
+    }
+
+    /// Representation of the user's relationship with the bank account holder.
+    public enum RelationshipType: String {
+        /// When the user owns the bank account
+        case `self` = "SELF"
+        /// When the bank account is owned by the user's business
+        case ownCompany = "OWN_COMPANY"
+    }
+
+    /// Representation of the shipping method for paper check
+    public enum ShippingMethodType: String {
+        /// Shipped in 2-15 business days
+        case standard = "STANDARD"
+        /// Shipped in 2-5 business days
+        case expedited = "EXPEDITED"
+    }
+
+    /// Representation of the bank account holder's role in the organization
+    public enum BusinessContactRole: String {
+        /// The director role
+        case director = "DIRECTOR"
+        /// The owner role
+        case owner = "OWNER"
+        /// The other role
+        case other = "OTHER"
+    }
+
+    /// Representation of the bank account holder's business type.
+    public enum BusinessType: String {
+        /// The corporation business type
+        case corporation = "CORPORATION"
+        /// The partnership business type
+        case partnership = "PARTNERSHIP"
```

Comment:
There are more in API doc https://docs.hyperwallet.com/content/api/v3/resources/paper-checks/list

businessTypestring | The bank account holder's business type. Possible types are:CORPORATION (default)PRIVATE_COMPANYPARTNERSHIPNOT_FOR_PROFIT_ORGANIZATIONGOVERNMENT_ENTITYPUBLIC_COMPANY
-- | --

Reply:
Just had a discussion with @wmews-hw . To avoid parsing issue similar to https://engineering.paypalcorp.com/jira/browse/DTSERWTHRE-305 let's not use enums anymore for the values that are not used in any logic in UI SDK.

Reply:
Good catch! I will update The bank account class as well. Also, docs is missing `CANADIAN_REGISTERED_CHARITY` as per https://github.com/hyperwallet/hw2/blob/master/rest-services/src/main/java/com/hw2/restservices/user/EBusinessType.java - are you aware of this?

Reply:
Hmm.. so we can get rid of all the enums in that class and use Strings instead? Except `ProfileType` ?